src/main/java/com/rackspace/salus/telemetry/ambassador/services/TestMonitorEventListener.java

```diff
@@ -53,6 +59,10 @@ public TestMonitorEventListener(EnvoyRegistry envoyRegistry,
     this.monitorApi = monitorApi;
     this.appName = appName;
     this.ourHostName = ourHostName;
+
+    requestsCounter = meterRegistry.counter("testMonitorListener.requests");
+    missingAgentCounter = meterRegistry.counter("testMonitorListener.missingAgent");
+    failedTranslateCounter = meterRegistry.counter("testMonitorListener.failedTranslate");
```

Comment:
```suggestion
    missingAgentCounter = meterRegistry.counter("testMonitorListenerErrors", "missingAgent");
    failedTranslateCounter = meterRegistry.counter("testMonitorListenerErrors", "failedTranslate");
```

Wouldn't making use of tags work better here?